### PR TITLE
Add Vidyard

### PR DESCRIPTION
**Pull request by xnumad**

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Is the "Get Help" button only available when you're logged in? Looking through that page, I don't see it.

**Comment by xnumad** [comment]
It is visible in the bottom right-hand corner. Additionally, the "Submit a request" link in the header achieves the same action.

**Comment by xnumad** [comment]
> What information do they require to request account deletion?

I just provided the email address of the account. I couldn't login on their [support page](https://knowledge.vidyard.com), only on their [main page](https://vidyard.com) (apparently using different accounts) so I submitted my request without being logged in. In the whole process, they didn't even verify the request with a confirmation I'd have to send from my email address meaning I could have deleted someone else's account by providing their email address.
> What follow-up occurs to that request?

Two mails from their side:
1. acknowledgment of receipt
2. deletion confirmation

Overall, the request was fulfilled within 48 hours. After that, a login returned `Incorrect username or password`, so I assume it has been fully deleted.

File: _data/sites.json [hunk 1]
```diff
@@ -6670,6 +6670,16 @@
         ]
     },
 
+    {
+        "name": "Vidyard",
+        "url": "https://knowledge.vidyard.com",
+        "difficulty": "medium",
+        "notes": "Click \"Get Help\" and request an account deletion.",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "notes": "Click \"Get Help\" and request an account deletion, entering your username into the request form.",
```

**Comment by Nightfirecat** [comment]
Thank you for your contribution, and for responding to feedback so promptly!

**Pull request merged**